src/models/mobileservices/keycloakrealmcr.js

```diff
@@ -90,6 +87,15 @@ export class KeycloakRealmCR extends CustomResource {
           comment: 'This is the set of rules that will be used to validate IDM bindings',
           fields: {
             realmSettings: {
+              realmId: {
+                validation_rules: [
+                  {
+                    type: 'maxlength',
+                    length: 36,
+                    error: 'realm id is too big, max length is 36 chars'
```

Comment:
There is no point in hardcoding the message here. Better to remove the error message here and just use the default one (see previous comment)